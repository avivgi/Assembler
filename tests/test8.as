.define len=2
 LIST:   .data   6, -9, len
MAIN:           mov     r3, LIST[2]