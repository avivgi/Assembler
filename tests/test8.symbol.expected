Symbol name: len       	Value: 2	type: 1: MDEFINE
Symbol name: LIST      	Value: 104	type: 3: DATA
Symbol name: MAIN      	Value: 100	type: 2: CODE
